goosebit/settings/schema.py

```diff
@@ -61,9 +64,25 @@ def settings_customise_sources(
         dotenv_settings: PydanticBaseSettingsSource,
         file_secret_settings: PydanticBaseSettingsSource,
     ) -> tuple[PydanticBaseSettingsSource, ...]:
-        return (
-            init_settings,
-            YamlConfigSettingsSource(settings_cls, BASE_DIR.joinpath("settings.yaml")),
-            env_settings,
-            file_secret_settings,
+        settings_sources = [env_settings]
+        config_files = []
+
+        if (path := os.getenv("GOOSEBIT_SETTINGS")) is not None:
+            config_files.append(Path(path))
+
+        config_files.extend(
+            [
+                CURRENT_DIR.joinpath("goosebit.yaml"),
+                GOOSEBIT_ROOT_DIR.joinpath("goosebit.yaml"),
+                Path("/etc/goosebit.yaml"),
+            ]
         )
+
+        for config_file in config_files:
+            if config_file.exists():
+                settings_sources.append(
+                    YamlConfigSettingsSource(settings_cls, config_file),
+                )
+                logger.info(f"Selected config file to load: {config_file}")
```

Comment:
This doesn't seem to work. I think at this point we cannot log yet.

Reply:
Yeah, this is run before the logger is instantiated.  Might need a new PR to modify the way the logger works, move it to another file so it can just log that after the logger is loaded.

Reply:
OK. In the meantime, I would just remove this and then this PR is good to go, I guess.

Reply:
I actually just added it, didn't notice that message.  I can remove it if you want, but its a good look at what needs to be done to accomplish it.